x-pack/legacy/plugins/ml/public/application/explorer/explorer_utils.d.ts

```diff
@@ -37,7 +36,7 @@ export declare const getDefaultSwimlaneData: () => SwimlaneData;
 export declare const getInfluencers: (selectedJobs: any[]) => string[];
 
 export declare const getSelectionInfluencers: (
-  selectedCells: SelectedCells,
+  selectedCells: AppStateSelectedCells | undefined,
```

Comment:
Tried to change it but then noticed: The cases where we have `AppStateSelectedCells | undefined` are not for interfaces but function arguments. There we cannot do `selectedCells?: ...` because for functions TS says: `A required parameter cannot follow an optional parameter`.

Reply:
oh yes, i missed that. i thought they were all interfaces.